packages/snap/src/index.ts

```diff
@@ -97,12 +99,49 @@ export const onRpcRequest = async ({
       return result;
     }
 
+    case RPCMethods.RemoveCredentials: {
+      const identityStorage = await getItemFromStore(StorageKeys.identity);
+      if (!identityStorage) {
+        throw new Error('Identity not created');
+      }
+
+      const params = request.params as RemoveCredentialsRequestParams;
+
+      const claimIds =
+        params?.claimIds || params?.ids.map((id) => getClaimIdFromVCId(id));
+
+      const vcManager = await VCManager.create();
+
+      const vcs = await vcManager.getDecryptedVCsByClaimIds(claimIds);
+
+      const dialogContent = [
+        heading('Remove Credentials'),
+        divider(),
+        ...vcs.map((el) => text(`${el.type[1]}`)), // FIXME
```

Comment:
Show how it looks like(screenshot)

Reply:
<img width="356" alt="image" src="https://github.com/rarimo/rarime/assets/39623143/8b173428-2015-468b-a4e3-56b98f0d54f6">

Reply:
should we have id here and separate multiple credentials in the proper way?
@ihordiachenko

Reply:
@ardier16 yes, let's do that